assets/stylesheets/base/_typography.scss

```diff
@@ -0,0 +1,56 @@
+$base-font-face: "acumin-pro", helvetica, arial, sans-serif;
+$condensed-font-face: "acumin-pro-extra-condensed", sans-serif;
+
+$base-font-size: 14px;
+
+@mixin font-style($face: $base-font-face, $size: $base-font-size, $weight: normal) {
```

Comment:
I think we need to be extra careful w/ naming conventions here... `font-style` is super generic. Why did you choose to prefix your selectors w/ `crds-*` and not handle mixins the same way?